### DISCUSSION TITLE
[BUG] Hard to see sync status icon in iOS app

### DISCUSSION THREAD

**Discussion by truppelito** [immich-app/immich#4787] [Feature Request]
### The bug

The sync icon on the iOS app is entirely white. If the photo/video behind it is also mostly white, the icon is very hard (sometimes impossible) to see:

![Untitled](https://github.com/immich-app/immich/assets/1059798/c1b48306-19db-41ae-a5e6-fbf79f97635e)

My suggestion is to add a small black drop shadow to the icon so it can be viewed over white backgrounds. My guess is this is probably quite simple and requires very little work. Another possibility is to detect if the item is white and use a black version of the icon instead (this probably requires more effort to implement). I don't know how these options will look, but the first option seems more general, because the second option would struggle with edge cases like an half-white/half-black image where the color boundary is below the icon.

### The OS that Immich Server is running on

Raspberry Pi OS

### Version of Immich Server

1.83.0

### Version of Immich Mobile App

1.83.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Standard content
```

### Your .env content

```Shell
Also standard content
```

### Reproduction steps

```bash
Not applicable.
```

**Comment by upsetdog** [comment]
Also applies for the stacked number and text too, with it being hard to see on white images. I think a drop shadow makes the most sense.
